contracts/StandardBounty.sol

```diff
@@ -0,0 +1,319 @@
+pragma solidity ^0.4.18;
+import "./inherited/StandardToken.sol";
+
+/// @title StandardBounties
+/// @dev Used to pay out individuals or groups for task fulfillment
+
+contract StandardBounty {
+
+  /*
+   * Storage
+   */
+
+  address public masterCopy;
+
+  address public controller;
+  string public data;
```

Comment:
I have to say - I love this approach...  So much better and cleaner.  Also, by having this inheritance pattern, we can make things much more distributed in the future which is cool.  This is def. going to require a decent refactor on the api side, and going to require a lot of thought.

Reply:
worth putting up an issue - to start discussion, share research etc.

Reply:
issue - internal contracts.